src/test/scala/com/exasol/cloudetl/bucket/AbstractBucketTest.scala

```diff
@@ -45,4 +45,25 @@ class AbstractBucketTest extends AnyFunSuite with BeforeAndAfterEach with Mockit
     when(metadata.getConnection("connection_info")).thenReturn(connectionInfo)
     metadata
   }
+
+  final def assertNoConnectionName(fn: => Unit): Unit = {
+    val thrown = intercept[BucketValidationException] {
+      fn
+    }
+    val msg = thrown.getMessage()
```

Comment:
in Java we don't use abbreviations. Not sure if it is ok here.

Reply:
It is fine here since it is test code. But I changed it.